runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -219,6 +230,99 @@ public void onWidgetLoaded(WidgetLoaded event)
 		}
 	}
 
+	@Subscribe
+	public void itemContainerChanged(ItemContainerChanged event)
+	{
+		final ItemContainer container = event.getItemContainer();
+		observeForWintertodtLootEntry(container);
+	}
+
+	private void observeForWintertodtLootEntry(ItemContainer container)
+	{
+		if (container != client.getItemContainer(InventoryID.INVENTORY))
+		{
+			return;
+		}
+
+		final Item[] items = container.getItems();
+
+		// EXTRA_SUPPLY_CRATE is the reward when you trade in items to Ignisia
+		long numberOfSupplyCrates = countSupplyCrates(items);
+
+		HashMultiset<Integer> currentSnapshot = generateInventorySnapshot(items);
+
+		// We want to make sure that we actually opened the supply crate, not banked it
+		if (numberOfSupplyCrates != lastNumberOfSupplyCrates && inventorySnapshot != null)
+		{
+			log.debug("Number of supply crates in inventory has changed from {} to {}", lastNumberOfSupplyCrates, numberOfSupplyCrates);
+
+			// we lost a crate by opening it
+			if (numberOfSupplyCrates < lastNumberOfSupplyCrates && !bankWidgetIsOpen() && !depositWidgetIsOpen())
```

Comment:
This still looks incredibly prone to false positives due to the fact I can deposit a crate by using it directly on a deposit box, thus never opening an widget

Reply:
I didn't even know you could do that to be honest

Reply:
I could change the code so that we only add entries when we don't have an empty gain of items, this would also fix a false entry if you die with the box. Of course, if you somehow managed to pick up a wintertodt loot item and bank the box at the same time via using on deposit box, that will still be a "loot" entry but that might be more trouble than it's worth to remedy.

Reply:
There's no realistic way to prevent that edge case, but in reality that's only going to happen in a minuscule amount of cases; or if you're intentionally going out of your way to produce that functionality.

Discarding empty loot events should sufficiently remedy false positives.